lightning/src/onion_message/offers.rs

```diff
@@ -117,6 +118,9 @@ impl OnionMessageContents for OffersMessage {
 			OffersMessage::InvoiceError(_) => INVOICE_ERROR_TLV_TYPE,
 		}
 	}
+	fn msg_type(&self) -> &'static str {
+		"Offers"
```

Comment:
We'll want to make a different one for each variant now.

Reply:
Addressed in [b4204b7](https://github.com/lightningdevkit/rust-lightning/commit/b4204b77c2417a5ca337dc2c336ae5d0871d421e)
Thanks for the pointer!